bundles/af-core/src/test/resources/form/image/exporter-image-parsedSrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "image-7cfd7f1fe4",
+  "fieldType": "image",
+  "name": "abcd",
+  "visible": false,
+  "altText": "abc",
+  "value": "/content/image-customized.img.png",
```

Comment:
Why are both value same, ideally they should be different ? What is difference between fileReference and value. Can you check how WCM Image component solve this